code/modules/mob/living/carbon/xenomorph/castes/ravager/abilities_ravager.dm

```diff
@@ -228,7 +228,8 @@
 	ADD_TRAIT(X, TRAIT_STAGGERIMMUNE, ENDURE_TRAIT) //Can now endure impacts/damages that would make lesser xenos flinch
 	ADD_TRAIT(X, TRAIT_SLOWDOWNIMMUNE, ENDURE_TRAIT) //Can now endure slowdown
 
-	RegisterSignal(X, COMSIG_XENOMORPH_TAKING_DAMAGE, PROC_REF(damage_taken)) //Warns us if our health is critically low
+	RegisterSignal(X, COMSIG_XENOMORPH_BRUTE_DAMAGE, PROC_REF(damage_taken)) //Warns us if our health is critically low
```

Comment:
Why split into two signals?

Reply:
For checking if you heal out of crit hp, it clears the visual. for instance if you're constantly going in and out of crit due to primo heal/super rage vamp. There is no single signal that I could find that could do this.

Reply:
What? Look at where the signals are sent. The general taking damage signal already encompasses both cases

Reply:
the xeno heals dont use the apply damage proc and only directly adjust burn and brute, so the xeno taking damage signal isnt used by them

Reply:
What a nightmare